console/lockable_trait.rst

```diff
@@ -43,4 +43,23 @@ that adds two convenient methods to lock and release commands::
         }
     }
 
+By default, the LockableTrait will use the ``FlockStore``, or the ``SemaphoreStore`` if available,
```

Comment:
This part is not fully clear to me:

```rst
will use the ``FlockStore``, or the ``SemaphoreStore`` if available [...]
```

I propose to reword it to better explain which of the two stores will be tried first:

```rst
The LockableTrait will use the ``SemaphoreStore`` if available and will default
to ``FlockStore`` otherwise. You can override this behavior [...]
```

Reply:
Thanks for the feedback, it's changed.